data/shwetasng.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Shweta Singh",
+  "displayStatsPublic":true,
+  "type":"personal",
+  "bio":"student"
+  "avatar": "https://github.com/shwetasng.png"
```

Comment:
It is not required to include your avatar, it will auto imported from github.

Reply:
```suggestion
```